### PR TITLE
202151013_Akash_Maurya

### PR DESCRIPTION
**Pull request by akashma2209**
update data.jason

### CODE REVIEW

File: ..txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ {
+      "name": {Akash Maurua},
```
Comment (nlok5923) [inline ..txt]: remove this file.

File: data.json [hunk 2]
```diff
@@ -9,5 +9,15 @@
       "linkedin": "https://www.linkedin.com/in/rahulbansal52/",
       "quote": "Open source is 💓."
     }
+  ]
+       "data": [
+      "name": {Akash Maurua},
+      "batch": {YOUR_BATCH_COMMENCEMENT_YEAR},
```
Comment (nlok5923) [inline data.json]: Please learn about `JSON` format. you can refer [this](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON)

**Review by nlok5923 — CHANGES_REQUESTED** [review]

**Comment by akashma2209** [comment]
Update data.json

**Comment by nlok5923** [comment]
The changes done by you aren't correct PTAL.

**Comment by akashma2209** [comment]
update data.json

**Comment by akashma2209** [comment]
update data.json

**Comment by nlok5923** [comment]
Please remove the unnecessary `..txt` file and please format the code properly in `data.json`

**Comment by akashma2209** [comment]
Update data.json

File: data.json [hunk 3]
```diff
@@ -9,5 +8,16 @@
       "linkedin": "https://www.linkedin.com/in/rahulbansal52/",
       "quote": "Open source is 💓."
     }
-  ]
-}
+
+
```
Comment (nlok5923) [inline data.json]:
```suggestion
```

File: data.json [hunk 4]
```diff
@@ -9,5 +8,16 @@
       "linkedin": "https://www.linkedin.com/in/rahulbansal52/",
       "quote": "Open source is 💓."
     }
-  ]
-}
+
+  
+        {
```
Comment (nlok5923) [inline data.json]:
```suggestion
   {
```

File: data.json [hunk 5]
```diff
@@ -1,5 +1,4 @@
-{
```
Comment (nlok5923) [inline data.json:1]: Also please don't delete the existing code.

**Review by nlok5923 — CHANGES_REQUESTED** [review]
Please address all the suggestions.

**Comment by akashma2209** [comment]
update data.json

File: data.json [hunk 6]
```diff
@@ -9,5 +9,14 @@
       "linkedin": "https://www.linkedin.com/in/rahulbansal52/",
       "quote": "Open source is 💓."
     }
```
Comment (nlok5923) [inline data.json]:
```suggestion
    },
```

**Review by nlok5923 — CHANGES_REQUESTED** [review]

**Pull request merged**